ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -33,40 +31,18 @@ public NotificationRepository(NtbsContext context)
             this.context = context;
         }
 
-        public async Task<IList<Notification>> GetRecentNotificationsAsync(IEnumerable<TBService> TBServices)
+        public async Task<IEnumerable<Notification>> GetRecentNotificationsAsync()
```

Comment:
The filtering logic by service/phec is now happening in Authorizationservice, as it's related to the user permissions, so the repo shouldn't have to worry about it